### PR TITLE
Rafactored ListItem in ChatListingPageView to be a component

### PR DESCRIPTION
**Pull request by pxlin-09**

### CODE REVIEW

File: android/app/src/main/java/org/uwaterloo/subletr/pages/chat/components/ChatListItem.kt [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.tooling.preview.Preview
+import org.uwaterloo.subletr.R
+import org.uwaterloo.subletr.pages.chat.Contact
+import org.uwaterloo.subletr.theme.SubletrTheme
+import org.uwaterloo.subletr.theme.subletrPalette
+
+
+@Composable
+fun ChatListItem(contact: Contact) {
```
Comment (leeminkyu16) [inline android/app/src/main/java/org/uwaterloo/subletr/pages/chat/components/ChatListItem.kt]: Can you add a `modifier` param to this?

**Review by leeminkyu16 — COMMENTED** [review]
Just one small comment.
I think otherwise, we'll be good

**Comment by pxlin-09** [comment]
> Just one small comment. I think otherwise, we'll be good

done

**Review by leeminkyu16 — APPROVED** [review]
Let's Gamble, Try Merging

**Review by abhaykshar — APPROVED** [review]
nice!

**Pull request merged**